src/io.rs

```diff
@@ -353,6 +353,8 @@ impl ZkIo {
                     Ok(sock) => sock,
                     Err(e) => {
                         error!("Failed to connect {:?}: {:?}", host, e);
+                        //retry too fast, sleep 1s
+                        sleep(Duration::from_secs(1)).await;
```

Comment:
This should be configurable, ideally with an exponential backoff.

Reply:
Add function connect_with with arg retry_time